capabilityreceiver/cmd/migratevalidations/main.go

```diff
@@ -42,40 +41,18 @@ type validationArgs struct {
 	capStatByte       []byte
 	tlsVersion        string
 	mimeTypes         []string
-	httpResponse      int
-	smartHttpResponse int
+	smartResponseByte []byte
 }
 
 // prepares the statements used in the "up" migration
 func prepareUpStatements(s *postgresql.Store) error {
 	var err error
 
-	addValidationResultStatement, err = s.DB.Prepare(`
-		INSERT INTO validation_results (id)
-		VALUES (DEFAULT)
-		RETURNING id;`)
-	if err != nil {
-		return err
-	}
-	addValidationStatement, err = s.DB.Prepare(`
-	INSERT INTO validations (
-		rule_name,
-		valid,
-		expected,
-		actual,
-		comment,
-		reference,
-		implementation_guide,
-		validation_result_id)
-	VALUES ($1, $2, $3, $4, $5, $6, $7, $8)`)
-	if err != nil {
-		return err
-	}
 	updateInfoValResStatement, err = s.DB.Prepare(`
 		UPDATE fhir_endpoints_info
 		SET
 			validation_result_id = $1
-		WHERE updated_at = $2 AND url = $3;`)
+		WHERE url = $2;`)
```

Comment:
Might want to add a @TODO here since the way the fhir_endpoint_info entry is found may need to change when the versions branch is merged in